src/archive.rs

```diff
@@ -320,6 +324,7 @@ impl<'a> EntriesFields<'a> {
                     ));
                 }
                 pax_extensions = Some(EntryFields::from(entry).read_all()?);
+                pax_size = pax_extensions_size(&pax_extensions.as_ref().unwrap());
```

Comment:
Could the `.unwrap()` be avoided here by parsing before storing the raw data in the `pax_extensions` local?